pdudaemon/dbhandler.py

```diff
@@ -61,7 +61,7 @@ def create_db(self):
 
     def insert_request(self, hostname, port, request, exectime):
         sql = "insert into pdu_queue (hostname,port,request,exectime) " \
-              "values ('%s',%i,'%s',%i)" % (hostname, port, request, exectime)
+              "values ('%s',%i,'%s',%i)" % (hostname, int(port), request, int(exectime))
```

Comment:
Sort of, the HTTP library returns everything as a string. Could have fixed it in the HTTP class but felt better to always cast to int before the write, in case anything else tries the same.